app/Models/Organization.php

```diff
@@ -26,6 +27,35 @@ class Organization extends Model
         'active',
     ];
 
+    /**
+     * Create a new organization and related records.
+     *
+     * @param string $orgName
+     * @param integer $userId
+     * @return boolean
+     */
+    public function createNewOrg(string $orgName, int $userId): bool 
```

Comment:
why not put this in the Org controller?

Reply:
because there may be other places we want to create orgs from so I felt like this method best lived inside of the model and that way the controllers can be less bloated later on.